app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,7 @@
     <%= csrf_meta_tags %>
     <%= action_cable_meta_tag %>
     <%= stylesheet_link_tag 'application', media: 'all' %>
+    <script src="https://unpkg.com/scrollreveal@4.0.0/dist/scrollreveal.min.js"></script>
```

Comment:
Please favor yarn adding librairies over using CDNs

Reply:
The guide did not instruct to add the package via yarn https://github.com/scrollreveal/scrollreveal